bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Voice.java

```diff
@@ -157,7 +158,7 @@ public static String interpret(@ParamDoc(name = "text") Object text) {
      * In case of interpretation error, the error message is played using the default audio sink.
      *
      * @param text The text to interpret
-     * @param interpreter The Human Language Interpreter to be used
+     * @param interpreter Comma separated list of human language text interpreters to use or null to use the default services
```

Comment:
Please rename the parameter into `interpreters`
This parameter should remain non null as you have another action without this parameter. So please remove the last part of the description.